core/src/scene/scene.h

```diff
@@ -10,10 +10,13 @@
 #include <vector>
 #include <unordered_map>
 
+#include "glm/glm.hpp"
```

Comment:
I would usually include just `glm/vec2.hpp` if I'm only using that type - not sure how much of a difference it makes on compile times but it can't hurt.